tests/translator/input/streams.yaml

```diff
@@ -25,3 +25,15 @@ Resources:
             Stream: arn:aws:dynamodb:us-west-2:012345678901:table/TestTable/stream/2015-05-11T21:21:33.291
             BatchSize: 200
             StartingPosition: LATEST
+  SQSFunction:
```

Comment:
Consider moving SQS test to a separate file. This file is already getting overloaded.